deploy/backplane/lpsre/acs/02-acs-lpsre-admins-project.ClusterRole.yaml

```diff
@@ -12,3 +12,12 @@ rules:
   - get
   - list
   - watch
+  - patch
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - pods/log
+  verbs:
+  - get
```

Comment:
typically see get/list/watch, any reason for only get?

Reply:
As far as I can tell make ran successfully. Checking. 

With limited namespace scope I don't think `list` is needed. `watch` is probably not a bad idea though.

Reply:
As far as I can tell make ran successfully. Checking. 

With limited namespace scope I don't think `list` is needed. `watch` is probably not a bad idea though.

Reply:
Output from build job indicates not all the changes done by `make` are committed then:

```
13:09:35 ERROR: uncommitted changes indicate generating content resulted in some file changes:
13:09:35 + git status --porcelain
13:09:35 + grep -v -e '.*sorted.*tmpl' -e '...hack.*'
13:09:35  M generated_deploy/backplane/lpsre/acs/02-acs-lpsre-admins-project.ClusterRole.yaml
```